frontend/src/lib/SessionInfo.test.ts

```diff
@@ -34,10 +34,14 @@ test("Clears session info", () => {
     commandLine: "command line",
     userMapboxToken: "mpt",
   })
+
   expect(SessionInfo.isSet()).toBe(true)
+  expect(SessionInfo.lastSessionId).toBe(undefined)
 
   SessionInfo.clearSession()
+
   expect(SessionInfo.isSet()).toBe(false)
+  expect(SessionInfo.lastSessionId).toBe("sessionId")
```

Comment:
Hm, will hold off on the `isSet` -> `hasSession` rename since that can probably be done later if decide to do so.

Aside from that, this comment should have been resolved by making the `lastSessionInfo` static suggestion from another comment